### PR TITLE
Added merge cells and fixed colours in Excel

### PR DESCRIPTION
**Pull request by greenenergy**
Great library! We just added merge cells and fixed the colours for MS Excel, since MS apparently doesn't follow their own spec.

### CODE REVIEW

**Comment by ShawnMilo** [comment]
@tealeg: Do you have an opinion on this? Tests pass. Nothing in the diff caught my eye, except the typo fixes. :o)

**Comment by tealeg** [comment]
@ShawnMilo Yup, it doesn't seem to be harmful.  There are some lines in the tests that have been commented out that should just be removed - I'll put an inline comment so you can see where I mean.

File: file_test.go [hunk 1]
```diff
@@ -649,8 +649,10 @@ func (l *FileSuite) TestMarshalFile(c *C) {
 	//
 	// For now we only allow simple string data in the
 	// spreadsheet.  Style support will follow.
+	// expectedStyles := `<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tealeg) [inline file_test.go]: Please remove these two commented lines.

File: xmlWorksheet.go [hunk 2]
```diff
@@ -311,5 +322,7 @@ func newXlsxWorksheet() (worksheet *xlsxWorksheet) {
 	worksheet.HeaderFooter.OddHeader[0] = xlsxOddHeader{Content: `&C&"Times New Roman,Regular"&12&A`}
 	worksheet.HeaderFooter.OddFooter = make([]xlsxOddFooter, 1)
 	worksheet.HeaderFooter.OddFooter[0] = xlsxOddFooter{Content: `&C&"Times New Roman,Regular"&12Page &P`}
+
+	//worksheet.MergeCells.Count = 0
```
Comment (tealeg) [inline xmlWorksheet.go]: Please remove this commented line.

**Comment by ShawnMilo** [comment]
@tealeg -- should we ask @greenenergy to make these changes and rebase, or should I just merge and make the changes myself? It doesn't matter functionally, but one way results in a nice commit log.

**Comment by tealeg** [comment]
@ShawnMilo I've tended to give the original author a window of opportunity to do it, and if nothing happened then I would perform a manual merge, putting my changes in on the way.

**Comment by ShawnMilo** [comment]
Okay. How do you do a manual merge? Do you add their fork as another remote locally, and merge the branch in?

I'll wait a few days or so before following up on this.

**Comment by tealeg** [comment]
@ShawnMilo I make  a new branch, the pull the pull request into it.  I then make my changes and commit them.  Finally I checkout my local master and merge in the branch I was working on and then push back to the repository.

**Comment by greenenergy** [comment]
Hey Guys. Sorry for not getting back sooner. I've made the requested changes.

**Pull request merged**